leakcanary-android/src/main/java/com/squareup/leakcanary/AndroidExcludedRefs.java

```diff
@@ -442,6 +443,13 @@
       excluded.instanceField("android.view.Choreographer$FrameDisplayEventReceiver",
           "mMessageQueue").alwaysExclude();
     }
+  },
+
+  INSTRUMENTATION_RECOMMEND_ACTIVITY(MEIZU.equals(MANUFACTURER)) {
```

Comment:
Can you move this closer to the top, next to the other manufacturer ignored leaks? Also please hide a version range check (SDK_INT)